src/Client/components/submission/code-editor.tsx

```
@@ -1,222 +1,277 @@
+            console.error("Error during code execution:", error);
+        }
+    };
+
+    return (
+        <div className="grid grid-cols-2 gap-4">
+            <Card className="border-none rounded-lg p-4 shadow-none bg-white dark:bg-[#18181b]">
+                <h2 className="text-xl font-bold mb-3">Problem Statement</h2>
+                <div className="space-y-4">
+                    <div>
+                        <p
+                            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)